File: src/main/java/ru/hlebozavod28/massak/CMLRunner/MassaKTestRead.java
package ru.hlebozavod28.massak.CMLRunner;

import lombok.extern.slf4j.Slf4j;
import org.springframework.boot.CommandLineRunner;
import org.springframework.stereotype.Component;

@Slf4j
@Component
public class MassaKTestRead implements CommandLineRunner{
    @Override
    public void run(String... args){
        log.info(System.getProperty("java.library.path"));
    }
}

File: src/main/java/ru/hlebozavod28/massak/CMLRunner/MassaKTestRead.kt
package ru.hlebozavod28.massak.CMLRunner

import org.springframework.boot.CommandLineRunner
import org.springframework.stereotype.Component
import java.util.logging.Logger

@Component
class MassaKTestRead : CommandLineRunner {
    private val log = Logger.getLogger(MassaKTestRead::class.java.name)
    override fun run(vararg args: String) {
//        log.info(System.getProperty("java.library.path"));
    }
}